code/modules/mob/living/brain/brain_item.dm

```diff
@@ -183,36 +190,49 @@
 	else
 		..()
 
-/obj/item/organ/brain/proc/get_brain_damage()
-	var/brain_damage_threshold = max_integrity * BRAIN_DAMAGE_INTEGRITY_MULTIPLIER
-	var/offset_integrity = obj_integrity - (max_integrity - brain_damage_threshold)
-	. = round((1 - (offset_integrity / brain_damage_threshold)) * BRAIN_DAMAGE_DEATH, DAMAGE_PRECISION)
-
-/obj/item/organ/brain/proc/adjust_brain_damage(amount, maximum)
-	var/adjusted_amount
-	if(amount >= 0 && maximum)
-		var/brainloss = get_brain_damage()
-		var/new_brainloss = CLAMP(brainloss + amount, 0, maximum)
-		if(brainloss > new_brainloss) //brainloss is over the cap already
-			return 0
-		adjusted_amount = new_brainloss - brainloss
-	else
-		adjusted_amount = amount
-
-	adjusted_amount = round(adjusted_amount * BRAIN_DAMAGE_INTEGRITY_MULTIPLIER, DAMAGE_PRECISION)
-	if(adjusted_amount)
-		if(adjusted_amount >= DAMAGE_PRECISION)
-			take_damage(adjusted_amount)
```

Comment:
Why don't you use the obj damage system for all organ damage instead of adding a redundant damage var?

Reply:
The organ damage system was modeled off of the liver damage system before it, since it paralleled the other damage mechanics the most. The liver damage system did not use the obj damage system, nor does the brain damage system. I'd be willing to make this change had this been introduced in the original PR about organ damage mechanics, but I don't want to change how it works again after the last one got finished.